tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java

```diff
@@ -46,4 +51,24 @@ private Long parseRunId(String id) {
             throw new IllegalArgumentException(e);
         }
     }
+
+    @Override
+    public TesServiceInfo getServiceInfo() {
+        String nameOfService = "CloudPipeline";
```

Comment:
move these values to application properties and inject them into service using `@Value` annotation